### PR TITLE
fix: add <minhaj-313>

### LINKED ISSUE #1785: [PROFILE ADDITION] <Shaikh Minhaj>
### Profile screenshot

minhaj-313 github profile

### PR DESCRIPTION
**Pull request by minhaj-313**
### 

resolves #1785

## Checklist for profile addition

- [X] Name the PR `fix: add <your-username>`.
- [X] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [X] You've linked the PR to the correct issue.

---

## Other additions/changes

- [X] Changes made in this PR are mentioned below
- [X] Search previous suggestions before making a new one, as yours may be a duplicate.
- [X] I acknowledge that all my contributions will be made under the project's license.
- [X] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/minhaj-313.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/107562768/229741200-dd444af0-23bb-4a1c-96c5-b43c53ef4273.png",
+    "issueId": 1785,
+    "name": "Shaikh Minhaj",
+    "username": "minhaj-313"
+}
```
Comment (Pradumnasaraf) [inline profiles/minhaj-313.json:6]:
```suggestion
}

```

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by minhaj-313** [comment]
@Pradumnasaraf @eddiejaoude  Hi, As mentioned On the Website, My Profile will be added to the organization after PR is Merged, but I have not received the invitation email yet from an Organization.

**Comment by eddiejaoude** [comment]
I am not sure what profile you are referring to, as there are more than one. Please follow the getting started guide in our Github Org here and any questions you can ask in the help channel in EddieHub Discord http://github.com/EddieHubCommunity?view_as=public

**Comment by minhaj-313** [comment]
Join the EddieHub GitHub Org by raising an [issue](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will receive an invitation)

@eddiejaoude 
 In the above line of Organisation Description you says by by raising an [issue](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will receive an invitation) but after merging my PR in EddieHubCommunity/awesome-github-profiles  I think it seems 2 weeks but till I not get invitation please send me invitation

**Pull request merged**